packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -210,6 +225,13 @@ class ScrollableBase extends Component {
 	indexToFocus = null
 	nodeToFocus = null
 
+	// overscroll
+	overscrollRefs = {['horizontal']: null, ['vertical']: null}
```

Comment:
It looks better to use `overscrollRefs = {horizontal: null, vertical: null}`.

Reply:
Right.